.travis.yml

```diff
@@ -3,10 +3,14 @@ cache: pip
 os:
 - linux
 python:
-  - "2.7"
-  - "3.6"
+  - 2.7
```

Comment:
We're supporting Python 2.7?

Reply:
Yes. But basically at zero cost to us; Cython handles it all automatically. We can drop it in a future release, or say it is "not officially supported".

But also, I thought we decided that we wanted Python 2 compatibility (well, maybe its been about a year ago now...).